src/components/App.css

```diff
@@ -16,6 +16,7 @@
 .editor-container {
   display: flex;
   flex: 1;
+  width: 100%;
```

Comment:
`flex` containers are like `block` they instantly are just 100% of the parent container in width. This rule is redundant. However, the deeper problem with the off rendering is the fact that this editor container is a flex display. Undoing `display: flex; and flex: 1;` both solve the width problem observed in Firefox.

This looks like one of the quirky cross-browser issues when it comes to flexbox. I'll try to make a reproducible sample from this to reference against the spec and file bugs where appropriate with.